tools/checkdocs/checkdocs.go

```diff
@@ -43,10 +43,10 @@ func main() {
 	}
 }
 
-// issueRE represents FerretDB issue url with label in Markdown format,
+// issueRE represents FerretDB issue or microsoft/document issue url with label in Markdown format,
 // such as: `[Label](https://github.com/FerretDB/FerretDB/issues/1)`.
-// It returns url as a submatch.
-var issueRE = regexp.MustCompile(`\[\w+]\((\Qhttps://github.com/FerretDB/\E[-\w]+/issues/\d+)\)`)
+// It returns url and owner/repo name as submatches.
+var issueRE = regexp.MustCompile(`\[\w+]\((\Qhttps://github.com/\E(FerretDB/[-\w]+|microsoft/documentdb)/issues/\d+)\)`)
```

Comment:
Do we want to check just the simple url? At the moment, it checks the following format only
```
[Label](https://github.com/FerretDB/FerretDB/issues/1)
```

Reply:
Yes, because we want to check URLs in comments, for example

Reply:
Updated to check simple URLs.